### PR TITLE
picview: Add version 2.2.7

### PR DESCRIPTION
**Pull request by Ruben2776**
Relates to #13388

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/picview.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Ruben2776/PicView/releases/download/2.2.7/PicView-v.2.2.7-win-x64.zip",
+            "hash": "b463c8b0795508932271f6c9de377fa13a96d1b96d482950d7fd702bc32ae3b3",
+            "extract_dir": ""
+        },
+        "arm64": {
+            "url": "https://github.com/Ruben2776/PicView/releases/download/2.2.7/PicView-v.2.2.7-win-arm64.zip",
+            "hash": "DD52E9916A38B442DB57B41B7A5737F09299455A9F2C14C4E095EC74998B360D",
+            "extract_dir": ""
```
Comment (aliesbelik) [inline bucket/picview.json:15]:
```suggestion
            "hash": "DD52E9916A38B442DB57B41B7A5737F09299455A9F2C14C4E095EC74998B360D"
```

File: bucket/picview.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+    "version": "2.2.7",
+    "description": "PicView is a fast, free and fully customizable image viewer for Windows 10 and 11.",
+    "homepage": "https://picview.org",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Ruben2776/PicView/releases/download/2.2.7/PicView-v.2.2.7-win-x64.zip",
+            "hash": "b463c8b0795508932271f6c9de377fa13a96d1b96d482950d7fd702bc32ae3b3",
+            "extract_dir": ""
```
Comment (aliesbelik) [inline bucket/picview.json:10]:
```suggestion
            "hash": "b463c8b0795508932271f6c9de377fa13a96d1b96d482950d7fd702bc32ae3b3"
```

File: bucket/picview.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+				"url": "https://github.com/Ruben2776/PicView/releases/download/$version/PicView-v.$version-win-x64.zip",
+                "extract_dir": ""
+            },
+            "arm64": {
+                "url": "https://github.com/Ruben2776/PicView/releases/download/$version/PicView-v.$version-win-arm64.zip",
+                "extract_dir": ""
```
Comment (aliesbelik) [inline bucket/picview.json:38]:
```suggestion
                "url": "https://github.com/Ruben2776/PicView/releases/download/$version/PicView-v.$version-win-arm64.zip"
```

File: bucket/picview.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    "checkver": {
+        "url": "https://api.github.com/repos/Ruben2776/PicView/releases/latest",
+        "jp": "$.tag_name",
+        "regex": "v([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+				"url": "https://github.com/Ruben2776/PicView/releases/download/$version/PicView-v.$version-win-x64.zip",
+                "extract_dir": ""
```
Comment (aliesbelik) [inline bucket/picview.json:34]:
```suggestion
				"url": "https://github.com/Ruben2776/PicView/releases/download/$version/PicView-v.$version-win-x64.zip"
```

**Review by aliesbelik — COMMENTED** [review]
`extract_dir` not necessary here and could be removed, both from download and autoupdate sections.

File: bucket/picview.json [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+            "PicView"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/Ruben2776/PicView"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+				"url": "https://github.com/Ruben2776/PicView/releases/download/$version/PicView-v.$version-win-x64.zip"
```
Comment (aliesbelik) [inline bucket/picview.json:29]:
```suggestion
                "url": "https://github.com/Ruben2776/PicView/releases/download/$version/PicView-v.$version-win-x64.zip"

```

**Review by aliesbelik — CHANGES_REQUESTED** [review]
> Error: [-] any leading whitespace consists only of spaces (excepting makefiles) 216ms (215ms|1ms)

File: bucket/picview.json [hunk 6]
```diff
@@ -0,0 +1,36 @@
+{
+    "version": "2.2.7",
+    "description": "PicView is a fast, free and fully customizable image viewer for Windows 10 and 11.",
```
Comment (niheaven) [inline bucket/picview.json:3]:
```suggestion
    "description": "A fast, free and fully customizable image viewer for Windows 10 and 11",

```

**Review by aliesbelik — APPROVED** [review]

**Pull request merged**